cips/cip-31.md

```diff
@@ -0,0 +1,157 @@
+
+| cip            | XX                                                                                                                      |
```

Comment:
re: filename, would suggest leaving un-numbered until this is ready for review.

nit: https://cips.celestia.org/cip-template.html
> Note: READ CIP-1 BEFORE USING THIS TEMPLATE! This is the suggested template for new CIPs. After you have filled in the requisite fields, please delete these comments. Note that an CIP number will be assigned by an editor. When opening a pull request to submit your CIP, please use an abbreviated title in the filename, cip-draft_title_abbrev.md. The title should be 44 characters or less. It should not repeat the CIP number in title, irrespective of the category.